manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -361,6 +361,25 @@ properties:
     links:
       passwd: (( concat "https://login." properties.system_domain "/forgot_password" ))
       signup: (( concat "https://login." properties.system_domain "/create_account" ))
+    oauth:
+      providers:
+        google:
```

Comment:
The script will delete the users and recreate them if it detects a change in origin, which is determined by what we call it. As Rafal points out we will then change the name (in our UAA configuration). However, UAA puts the name in the redirect URIs, and the redirect URIs have to be whitelisted in the Google developer console. This means manually updating in the console, but you can have multiple whitelistings for each set of OAuth credentials, so you can dual run the old and new names.